Gems/Atom/Asset/ImageProcessingAtom/Code/Source/Processing/ImageConvert.cpp

```diff
@@ -352,7 +354,8 @@ namespace ImageProcessingAtom
         // output conversion log
         if (m_isSucceed && m_isFinished)
         {
-            [[maybe_unused]] const uint32 sizeTotal = m_image->Get()->GetTextureMemory();
+            IImageObjectPtr imageObj = m_image->Get();
```

Comment:
Does this need a [[maybe_unused]] too? It looks like it's only being used with other maybe_unused variables.

Reply:
Good call, I'm guessing that'd trip on a release build